googleapis_auth/lib/src/oauth2_flows/token_model.dart

```diff
@@ -175,12 +179,14 @@ Future<CodeResponse> requestAuthorizationCode({
 Future<void> revokeConsent(String accessTokenValue) {
   final completer = Completer<void>();
 
-  void done(Object? arg) {
+  void done(JSAny? arg) {
     window.console.log(arg);
     completer.complete();
   }
 
-  gis.oauth2.revoke(accessTokenValue, allowInterop(done));
+  // TODO(srujzs): Once `TokenRevocationResponse` is an extension type that
```

Comment:
why are we not doing this now?

Reply:
That belongs to a separate repo, but that repo's SDK constraints are `^3.3.0` so I believe we could. I'm guessing no one got around to it yet.

Reply:
Oh! I bet that's Davids auth thingy...